safe-global/safe-dao-governance-app
# feat: migrate to WalletConnect v2

## What it solves

Partially resolves #27

## How this PR fixes it

This upgrades onboard and adds the WalletConnect v2 module, mirroring [the implementation in `safe-wallet-web`](https://github.com/safe-global/safe-wallet-web/issues/2017).

## How to test it

Open the app with an EOA and observe onboard still working, with the presence of WalletConnect v2.

## Checklist
